sark/code/function.py

```diff
@@ -181,6 +181,14 @@ def crefs_to(self):
         return idautils.CodeRefsTo(self.startEA, 1)
 
     @property
+    def calls(self):
+        """
+        Calls to other functions.
+        Returns: Xref objects
+        """
+        return ifilter(lambda x: x.type.is_call, self.xrefs_from)
```

Comment:
I'm not sure I understand what you mean.
if you meant it instead of the ifilter wouldn't it create a list?
I though you are in favor of iterators.